roll-numbers/mutation_2.py

```diff
@@ -0,0 +1,9 @@
+def assign_roll_numbers(students):
+    '''
+       	Incorrect roll number assignment
```

Comment:
Unnecessary extra indentation (im not sure if it will do anything, but just do it to make sure)